### ISSUE TITLE
BUG: Too Many Redirects

### ISSUE THREAD

**Issue by LulzLoL231** [k1nky/netbox-otp-plugin#6]
Hi, after installing plugin and logout from Netbox, browser can't connect to Netbox because looping redirect from `/login/?next=/plugins/otp` to `/plugins/otp` and again, calling browser error TOO_MANY_REDIRECTS.

Changing browser, re-installing did not help.  
Netbox version: 3.5.1
netbox-otp-plugin version: 1.0.5

<sub>Sorry for bad english, i'm not good at it</sub>

**Comment by k1nky** [comment]
Hi,

I tried to reproduce it but couldn't get this behavior. My test environment is Netbox 3.5.1 and netbox-otp-plugin 1.0.5. Do you have a proxy ahead your Netbox?

Could you describe your steps to reproduce it?

**Comment by andersonit** [comment]
I'm having the same issue. I have LOGIN_REQUIRE = True in configuration.py and using Netbox 3.5.3 with the latest Plugin version. The browser gives error: ERR_TOO_MANY_REDIRECTS when bringing up the login page.

**Comment by andersonit** [comment]
If it helps, this is what's showing in the nginx access.log.  You can see it keeps redirecting between "/plugins/otp" and "/login/?next=plugins/otp/" until the browser throws the error_too_many_redirects.  This only happens when "LOGIN_REQUIRED = True" in the configuration.py.  

"GET /login/?next=/plugins/otp/ HTTP/1.1" 302 0 "-"
"GET /plugins/otp/ HTTP/1.1" 302 0 "-" 
"GET /login/?next=/plugins/otp/ HTTP/1.1" 302 0 "-"
"GET /plugins/otp/ HTTP/1.1" 302 0 "-" 
"GET /login/?next=/plugins/otp/ HTTP/1.1" 302 0 "-"
"GET /plugins/otp/ HTTP/1.1" 302 0 "-" 
"GET /login/?next=/plugins/otp/ HTTP/1.1" 302 0 "-"
"GET /plugins/otp/ HTTP/1.1" 302 0 "-" 

I've tried looking through the code, but this is above my capabilities... give me some classic ASP and VBScript and I'd be fine LOL

**Comment by andersonit** [comment]
Sorry, I just saw you had updated the __init__.py file in the dev branch.  I updated my file with that new line and it's working now!  Thank you so much!

**Comment by k1nky** [comment]
@andersonit, thank you! LOGIN_REQUIRED was the reason for the circular redirect. 

```
 # Setting this to True will permit only authenticated users to access any part of NetBox. By default, anonymous users
 # are permitted to access most data in NetBox but not make any changes.
LOGIN_REQUIRED = True
```
So, when LOGIN_REQUIRED = True, a user must be authorized before accessing to /plugin/otp location. But a user can do it only in  /plugin/otp. :smile:

I have published a new version (1.0.7) of the package.
